src/components/RecipeCard/RecipeCard.jsx

```diff
@@ -46,8 +53,28 @@ export default function RecipeCard(props) {
     <div className="recipe-card">
         {/* recipe card */}
         {error && <Alert variant="danger">{error}</Alert>}
-        <img src={props.image} alt={`recipe for ${props.title}`} onClick={() => handleRecipeCardClick(true, props.id)}/>
-        <p className="recipe-name">{props.title}</p>
+        <img className="recipe-img" src={props.curRecipe.image} alt={`recipe for ${props.curRecipe.title}`} onClick={() => handleRecipeCardClick(true, props.curRecipe.id)}/>
+        <div className="recipe-info">
+          <div className="recipe-heading">
+            <div className="recipe-title">
+              <p className="recipe-cuisine">{cuisine.toUpperCase()}</p>
+              <p className="recipe-name">{props.curRecipe.title}</p>
+            </div>
+            <img className="heart-img" src={ heart } alt="heart icon" />
+          </div>
+          <div className="recipe-extra-info">
+            {validMins && 
+              <div className="recipe-time">
+                <img className="timer-img" src={ timer } alt="timer icon" /> <p className="time-num">{props.curRecipe.readyInMinutes} mins</p>
+              </div>
+            }
+            {validLikes &&
+              <div className="recipe-likes">
+                <p className="like-num">{props.curRecipe.aggregateLikes}</p> <img className="like-img" src={ like } alt="thumbs up icon" />
+              </div>
+            }
+          </div>
```

Comment:
Should the like, timer, heart buttons be made into their own components? That may make them easier to style and re-use as well as keeping this code simpler

Reply:
I agree, I will make the heart button an own component since later on, I need to add a onClick function on the heart icon, but the other icons I think I will only use in the Recipes page and not re-use anywhere else (and they wouldn't have their own functions) so it might be better to keep it as is